.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,36 +1,41 @@
-# Pull Request Template #
```

Comment:
Are we changing, in the future, this file also to `.yaml` format?

Reply:
Up to you all. If it makes it easier for the team to complete PRs with the required info, then let's do it. If they're ok with having the raw MD template with the hints as comments, then we can leave it.

Reply:
Let's keep it as it is for now and do it in a future task in we consider so.